### DISCUSSION TITLE
DOH client

### DISCUSSION THREAD

**Discussion by lsellens** [homeassistant-apps/app-cloudflared#556] [Ideas]
Would it be possible to make the option available to use this addon as a DOH client? I don't own a hostname an have no interest in purchasing one, but I use cloudflared as a doh client currently. I'm wanting to migrate my setup to hassio, but it doesn't look like this addon supports using it for doh. I think this would be a great addition to use along side adguard home since there is no unbound addon I could find.
https://developers.cloudflare.com/1.1.1.1/encryption/dns-over-https/dns-over-https-client/

**Comment by brenner-tobias** [comment]
Thanks a lot for raisining this. We discussed offering that possibility but decided against it, since there are already many possibilities to do this with other solutions and we do not want to blow-up this add-on too much. See discussion #265
So for now, I will close this as not going to happen.
